frontend/packages/core/src/AppLayout/stories/header.stories.tsx

```diff
@@ -0,0 +1,27 @@
+import * as React from "react";
+import { MemoryRouter } from "react-router";
+import type { Meta } from "@storybook/react";
+
+import { ApplicationContext } from "../../Contexts/app-context";
+import Header from "../header";
+
+export default {
+  title: "Core/AppLayout/Header",
+  component: Header,
+  decorators: [
```

Comment:
added these decorators b/c the Header imports components that make use of `Link` and `React.createContext`. without it, the Header story fails with errors such as `useHref() may be used only in the context of a <Router> component` and `Cannot read property 'workflows' of undefined`